Flatpak/org.dolphin_emu.dolphin_emu.metainfo.xml.in

```diff
@@ -1,11 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2016 Jeremy Newton -->
 <component type="desktop-application">
-  <id>org.DolphinEmu.dolphin-emu</id>
+  <id>org.dolphin_emu.dolphin_emu</id>
```

Comment:
I'm not sure if you can _just_ go and change this without breaking Flatpak (or something on FlatHub). At least I recall some discussions about the ID, and someone saying that there is a lot more things involved than just changing it in here.

But I think @OatmealDome can probably add more to this.